readme.md

```diff
@@ -45,6 +45,37 @@ Non-enumerable properties are kept non-enumerable (name, message, stack).
 Enumerable properties are kept enumerable (all properties besides the non-enumerable ones).
 Circular references are handled.
 
+#### options
+
+##### allowToJSON
+
+Type: `Boolean`, default: `false`
+
+If `true` and if the input object contains `.toJSON()` then its result will be returned instead of object properties.
+
+It's up on `.toJSON()` implementation to handle circular references and enumerability of the properties.
+
+Example:
```

Comment:
Although these examples are technically correct, I'd say they should use Error instances, as anything else is out of scope for this package, and probably falls into "undefined behavior" territory.